### PR TITLE
Zeebe Node Client 0.23.2 integration

### LINKED ISSUE #230: Migrate to Node Zeebe client 0.23.0
We should migrate to [Node Zeebe Client 0.23.0](https://github.com/creditsenseau/zeebe-client-node-js/releases/tag/v0.23.0) which contains a JS only implementation for gRPC operations. That'd help us limit the size of Zeebe Modeler build and also build time would be much faster.

### PR DESCRIPTION
**Pull request by oguzeroglu**
Closes #230 

With this PR:

1) Build time is back to normal (takes couple of seconds instead of 5 minutes)
2) The build size is back to normal (from 88 MB to 67 MB)
3) We're getting rid of electron-rebuild related tasks, since we no longer depend on the gRPC module.
4) Deployment names are now suffixed with `.bpmn`, since with this version we're getting this error from ZB for non-suffixed deployment names: `details: "Expected argument 'name' to be a string ending with either .bpmn or .yaml, but was XYZ"`

### CODE REVIEW

File: app/lib/zeebe-api.js [hunk 1]
```diff
@@ -247,3 +253,28 @@ function withoutSecrets(parameters) {
...
+// .bpmn suffix.
+//
+// If name is empty, we'll return the file name. If name is not empty
+// but does not end with .bpmn, we'll add the suffix.
+function prepareDeploymentName(name, filePath) {
+
+  try {
+
+    if (!name || name.length === 0) {
+      return path.basename(filePath);
```
Comment (barmac) [inline app/lib/zeebe-api.js:267]: This fails for `.xml` extension.

`Expected argument 'name' to be a string ending with either .bpmn or .yaml, but was zeebe.xml [ deploy-error ]`

Comment (barmac) [reply app/lib/zeebe-api.js:267]: How to reproduce:
1. Create and save `zeebe.xml` file.
2. Try to deploy with empty deployment name.

File: app/package.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "mri": "^1.1.4",
     "node-fetch": "^2.1.2",
     "parents": "^1.0.1",
-    "zeebe-node": "^0.22.1"
+    "zeebe-node": "^0.23.0"
```
Comment (barmac) [inline app/package.json:20]: Why not `0.23.2`?

File: app/test/spec/zeebe-api-spec.js [hunk 3]
```diff
@@ -269,6 +269,96 @@ describe('ZeebeAPI', function() {
...
+        ZBClient: function() {
+          return {
+            deployWorkflow: deployWorkflowSpy,
+          };
+        }
+      });
+
+      // when
+      await zbAPI.deploy({
+        filePath: '/Users/Test/Stuff/Zeebe/process.bpmn',
```
Comment (barmac) [inline app/test/spec/zeebe-api-spec.js:349]: Let's add a test case when filename does not end with `.bpmn` (e.g. `zeebe.xml`).

File: package.json [hunk 4]
```diff
@@ -18,9 +18,7 @@
     "build:test-distro": "node tasks/test-distro.js",
     "build": "run-s client:build \"build:distro -- {@}\" \"build:test-distro -- {@}\" --",
     "lint": "eslint .",
-    "postinstall": "run-s lerna-bootstrap rebuild-native",
-    "lerna-bootstrap": "lerna bootstrap",
-    "rebuild-native": "electron-builder install-app-deps",
```
Comment (barmac) [inline package.json:23]: 💕

File: resources/diagram/simple.bpmn [hunk 5]
```diff
@@ -1,54 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (barmac) [inline resources/diagram/simple.bpmn:1]: Let's remove this change, shall we?

**Comment by barmac** [comment]
I left a few comments which needs to be addressed before we merge this. Overall, it is a huge improvement :)

File: app/package.json (hunk 2 above)
Comment (oguzeroglu) [reply app/package.json]: 🐵

File: resources/diagram/simple.bpmn (hunk 5 above)
Comment (oguzeroglu) [reply resources/diagram/simple.bpmn]: 🐵

File: app/test/spec/zeebe-api-spec.js (hunk 3 above)
Comment (oguzeroglu) [reply app/test/spec/zeebe-api-spec.js:349]: Done here: https://github.com/zeebe-io/zeebe-modeler/blob/34234eb5b426267d2e1bc0431702e97e6672514e/app/test/spec/zeebe-api-spec.js#L364

File: app/lib/zeebe-api.js (hunk 1 above)
Comment (oguzeroglu) [reply app/lib/zeebe-api.js]: Nice catch ⭐

**Pull request merged**